content/docs/import/import-from-postgres.md

```diff
@@ -29,7 +29,7 @@ Repeat the `pg_dump` and `pg_restore` process for each database you want to migr
    </CodeBlock>
 
 - Consider running a test migration first to ensure your actual migration goes smoothly. See [Run a test migration](#run-a-test-migration).
-- If your database is small (< 1 GB), you can pipe `pg_dump` output directly to `pg_restore` to save time. See [Pipe pg_dump to pg_restore](#pipe-pg_dump-to-pg_restore).
+- If your database is small (< 1 GiB), you can pipe `pg_dump` output directly to `pg_restore` to save time. See [Pipe pg_dump to pg_restore](#pipe-pg_dump-to-pg_restore).
```

Comment:
FWIW i don't think it's critical to have GiB here.
the size the user will be getting from other systems, not neon

up to docs team to decide

Reply:
True.

Reply:
Removed GiB